File: Assignment 9/program2.c
////////////////////////////////////////////////////////////////////////////////
//
//  Required Heder Files
//
////////////////////////////////////////////////////////////////////////////////
#include<stdio.h>        //For Input Output
////////////////////////////////////////////////////////////////////////////////
//
//  Function Name : RangeDisplayEven
//  Description   : It is used to Display Even number
//  Input         : Intiger  
//  OutPut        : Intiger  
//  Author        : Pratibha Sagar Pawar
//  Date          : 31 Oct 2025
//      
////////////////////////////////////////////////////////////////////////////////
void RangeDisplayEven(int iStart, int iEnd)
{
    
    if(iStart > iEnd)
    {
        printf("Invalid range");
        return ;
    }

    int iCnt = 0;
    for(iCnt = iStart; iCnt<= iEnd ; iCnt++)
    {
        if(iCnt % 2 == 0)
        {
             printf("%d\t",iCnt);
        }
    }
}
////////////////////////////////////////////////////////////////////////////////
//
//  Entery point function for the application 
// 
////////////////////////////////////////////////////////////////////////////////
int main()
{
    int iValue1 =0,  iValue2 = 0;

    printf("Enter Starting point ");
    scanf("%d",&iValue1);

    printf("Enter Ending point ");
    scanf("%d",&iValue2);

    RangeDisplayEven(iValue1, iValue2);

    return 0;

}
//  End of main Function
////////////////////////////////////////////////////////////////////////////////
//
//  Input : 23 35     Output : 24 26 28 30 32 34
//  Input : 10 18     Output : 10 12 14 16 18
//  Inpur : 10 10     Output : 10
//
////////////////////////////////////////////////////////////////////////////////

File: Assignment 28/program2.java
////////////////////////////////////////////////////////////////////////////////
//
//  Required Package
//
////////////////////////////////////////////////////////////////////////////////

import java.util.*;

////////////////////////////////////////////////////////////////////////////////
//
//  Class Name    : class pattern
//  Description   : It is used to Display pattern 
//  Input         : Intiger  
//  OutPut        : Pattern 
//  Author        : Pratibha Sagar Pawar
//  Date          : 21 Nov 2025
//      
////////////////////////////////////////////////////////////////////////////////
class pattern
{
    public void Display(int iRow, int iCol)
    {
        int i = 0;
        int j = 0;
        char ch = '\0';

        for(i = 1; i<=iRow; i++)
        {
            for(j=1; j<=iCol; j++)
            {
                if(i % 2 == 0)
              {
                ch = (char)('a' + j-1);
               System.out.print(ch+"\t");
              }
              else
              {
                ch = (char)('A' + j-1);
               System.out.print(ch+"\t");
              }
              
            }
            System.out.println();
        }
         System.out.println();    
    
    }
    
}
///////////////////////////////////////////////////////////////////////////////
//
//  Entery point method for the application 
// 
////////////////////////////////////////////////////////////////////////////////
class program2
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0;
        int iValue2 = 0;

        System.out.println("Enter the  number of Row : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter the number of Column : ");
        iValue2 = sobj.nextInt();

        pattern pobj = new pattern();
        pobj.Display(iValue1,iValue2);
    }
}
//  End of main method
////////////////////////////////////////////////////////////////////////////////
//
//  Input  : iRow = 4  iCol = 4
//
//  Output : A   B   C   D
//           a   b   c   d
//           A   B   C   D
//           a   b   c   d
//
////////////////////////////////////////////////////////////////////////////////

